### DISCUSSION TITLE
Thanks for this wonderful commander, Alex ! & Double Commander Lite

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1114] [Suggestions]
### Description

Again many, many thanks Alex for your unique commander.
I find it of great use since feb-2022 !
It is my ONE & ONLY commander, besides windows file explorer (but i don't use windows file explorer at all).

However i had made a fork of yours unique commander and i made it Lite, since the main reason is to preserve it's touch.

Hope you give it a try and if you don't agree please do tell me and i will take the fork down ASAP !
https://github.com/nvdavsamd/doublecmdlite

Many many thanks !
With great respect and regards !

![doublecmd lite a](https://github.com/doublecmd/doublecmd/assets/122004809/ed1be5c0-9032-48d3-ab8c-81aa9c63556f)

**Comment by gh-log** [comment]
I would like Dark Mode for DC like yours!

**Comment by ghost** [comment]
Hi i updated again to be exactly like lite version !
Enjoy !
